src/main/apisix/values.yaml

```diff
@@ -0,0 +1,157 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+replicaCount: 1
+image:
+  repository: apache/apisix
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 1.3-alpine
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+gateway:
+  type: NodePort
+  # type: LoadBalancer
+  # annotations:
+  #   service.beta.kubernetes.io/aws-load-balancer-type: nlb
+  http:
+    enabled: true
+    servicePort: 80
+    containerPort: 9080
+  tls:
+    enabled: false
+    servicePort: 443
+    containerPort: 9443
+    http2:
+      enabled: true
+
+  ingress:
+    enabled: false
+    annotations:
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: apisix.local
+        paths: []
+    tls: []
+  #  - secretName: apisix-tls
+  #    hosts:
+  #      - chart-example.local
+
+# etcd configuration
+# use the FQDN address or the IP of the etcd
+etcd:
+  host: http://etcd-apisix.local:2379 # http://172.20.128.89:2379
+  prefix: "/apisix"
+  timeout: 3
+  enabled: true
+
+dns:
+  #resolvers:
+  #  - 127.0.0.1
+  #  - 172.20.0.10
+  #  - 114.114.114.114
+  #  - 223.5.5.5
+  #  - 1.1.1.1
+  #  - 8.8.8.8
+  validity: 30
+  timeout: 5
+
+admin:
+  # Enable Admin API
+  enabled: true
+  # Admin API support CORS response headers
+  cors: true
+  # Admin API credentials
+  credentials:
+    admin: edd1c9f034335f136f87ad84b625c8f1
+    viewer: 4054f7cf07e344346cd3f287985e76a2
+
+# APISIX plugins to be enabled
+plugins:
+  - example-plugin
+  - limit-req
+  - limit-count
+  - limit-conn
+  - key-auth
+  - basic-auth
+  - prometheus
+  - node-status
+  - jwt-auth
+  - zipkin
+  - ip-restriction
+  - grpc-transcode
+  - serverless-pre-function
+  - serverless-post-function
+  - openid-connect
+  - proxy-rewrite
+  - redirect
+  - response-rewrite
+  - fault-injection
+  - udp-logger
+  - wolf-rbac
+  - proxy-cache
+  - tcp-logger
+  - proxy-mirror
+  - kafka-logger
+  - cors
+stream_plugins:
+  - mqtt-proxy
+
+resources: 
+  limits:
+    cpu: '4'
+  requests:
+    cpu: '50m'
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
```

Comment:
Helm recommends not to specify `resources.limits` by default, and leave the choice for the users instead. We can only specify `resources.requests` here:

```
resources:
  limits:
    # cpu, memory
  requests:
    cpu: 100m
    memory: 128Mi
```

And don't forget etcd above.

Reply:
done